src/useSpeech/__docs__/story.mdx

```diff
@@ -0,0 +1,48 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Example } from './example.stories';
+import { ImportPath } from '../../__docs__/ImportPath';
+
+<Meta title="DOM/useSpeech" component={Example} />
```

Comment:
I'd dedicate it to navigator section as it has nothing to do with DOM

Reply:
Change is [here](https://github.com/react-hookz/web/pull/1378/commits/68a922849a696fd0db22c504031130534f509db0)